packages/streaming-image-volume-loader/src/ImageLoadRequests.ts

```diff
@@ -0,0 +1,30 @@
+import { Types, Enums } from '@cornerstonejs/core';
+
+export default interface ImageLoadRequests {
+  callLoadImage: (
+    imageId: any,
+    imageIdIndex: any,
+    options: any
+  ) => Promise<void>;
+  imageId: string;
+  imageIdIndex: number;
+  options: {
+    targetBuffer: {
+      arrayBuffer: SharedArrayBuffer;
```

Comment:
I think if we don't use the sharedArrayBuffer this is undefine so it is SharedArrayBuffer | undefined